### DISCUSSION TITLE
第6讲第三题 数组

### DISCUSSION THREAD

**Discussion by ztmanhattan** [AmazingAng/WTF-Solidity#464] [General]
以下关于数组的说法中，正确的是
*
固定长度数组 和 bytes拥有 push() 成员，可以在数组最后添加一个0元素。
数组字面常数，例如 [uint(1),2,3]，需要声明第一个元素的类型，不然默认用存储空间最大的类型。
内存数组的长度在创建后是固定的。
对于memory可变长度数组，可以用new操作符来创建，并且不用声明长度，例如uint[] memory array = new uint[];。

这四个选项是不是都是不正确的? 我无论怎么选自己的分数都是83%? 有大家遇到一样的情况吗

**Comment by Azleal** [comment]
可能你的是其他题目有错误的

**Comment by maixuanfeng1** [comment]
内存数组的长度在创建后是固定的。这一条是对的

**Comment by Naoki2015** [comment]
内存数组的长度在创建后是固定的。这一项是对的
